sources/api/host-containers/src/main.rs

```diff
@@ -202,23 +202,30 @@ impl<'a> SystemdUnit<'a> {
         }
     }
 
+    fn enable(&self) -> Result<()> {
+        command(SYSTEMCTL_BIN, &["enable", &self.unit])?;
+
+        Ok(())
+    }
+
     fn enable_and_start(&self) -> Result<()> {
-        // We enable/start units with --no-block to work around cyclic dependency issues at boot
-        // time.  It would probably be better to give systemd more of a chance to tell us that
-        // something failed to start, if dependencies can be resolved in another way.
         command(
             SYSTEMCTL_BIN,
-            &["enable", &self.unit, "--now", "--no-block"],
-        )
+            &["enable", &self.unit, "--now"]
```

Comment:
Before the refactor, this would hold up the boot because of a circular dependency - we couldn't start the container until containerd started, but containerd couldn't start until we were finished applying settings.

I expect it's better to keep `--no-block`, because nothing is waiting on this process and so it may as well exit quickly, and because it could cause trouble later if we start paying more attention to processes spawned by apiserver.

Reply:
I added `--no-block` to the command in the last forced push I did, as well as the `log-level` flag.